packages/block-library/src/post-time-to-read/block.json

```diff
@@ -45,6 +45,18 @@
 		},
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true,
+			"__experimentalDefaultControls": {
+				"radius": true,
+				"color": true,
+				"width": true,
+				"style": true
+			}
```

Comment:
There was [some feedback](https://github.com/WordPress/gutenberg/pull/63543#issuecomment-2232971361) when adding border support to other blocks that didn't display the dimension controls by default that the border controls should be optional. Perhaps we should follow that advice here as well?

Reply:
I have updated json